yb-voyager/cmd/archiveChangesCommand.go

```diff
@@ -43,9 +43,12 @@ var archiveChangesCmd = &cobra.Command{
 
 func archiveChangesCommandFn(cmd *cobra.Command, args []string) {
 	moveToChanged := cmd.Flags().Changed("move-to")
-	deleteChanged := cmd.Flags().Changed("delete")
+	deleteChanged := cmd.Flags().Changed("delete-changes-without-archiving")
```

Comment:
The `acrhive changes` command already indicates "changes", maybe we keep this flag just `delete-without-archiving` ? @amit-yb